gensim/summarization/bm25.py

```diff
@@ -271,7 +291,7 @@ def iter_bm25_bow(corpus, n_jobs=1):
     pool.join()
 
 
-def get_bm25_weights(corpus, n_jobs=1):
+def get_bm25_weights(corpus, n_jobs=1, is_bow=False):
```

Comment:
Please update the documentation to include the new parameter.

Also, please add unit tests for the new functionality.